my-account/app/com/commercetools/sunrise/myaccount/authentication/recoverpassword/reset/ResetPasswordFormData.java

```diff
@@ -0,0 +1,23 @@
+package com.commercetools.sunrise.myaccount.authentication.recoverpassword.reset;
+
+import com.google.inject.ImplementedBy;
+
+/**
+ * The data required to reset a customers password.
+ */
+@ImplementedBy(DefaultResetPasswordFormData.class)
+public interface ResetPasswordFormData {
+    /**
+     * The new password.
+     *
+     * @return the new password
+     */
+    String newPassword();
+
+    /**
+     * The confirm password.
+     *
+     * @return the repeated new password
+     */
+    String confirmPassword();
```

Comment:
Sorry to bring this up and not before, but do you think we should expose this in the interface? The idea is to have here only the "util" data coming from the form, not the raw data. The confirmation password is more an "implementation" detail, not useful for the operation. IMO we should remove it from the interface.